administrator/components/com_admin/sql/updates/mysql/6.0.0-2025-08-22.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO `#__guidedtours` (`title`, `description`, `extensions`, `url`, `published`, `language`, `note`, `access`, `uid`, `autostart`, `created`, `created_by`, `modified`, `modified_by`) VALUES
+('COM_GUIDEDTOURS_TOUR_WHATSNEW_6_0_TITLE', 'COM_GUIDEDTOURS_TOUR_WHATSNEW_6_0_DESCRIPTION', '["com_cpanel"]', 'administrator/index.php', 1, '*', '', 1, 'joomla-whatsnew-6-0', 1, CURRENT_TIMESTAMP(), 0, CURRENT_TIMESTAMP(), 0);
```

Comment:
```suggestion
INSERT INTO `#__guidedtours` (`title`, `description`, `extensions`, `url`, `published`, `language`, `note`, `access`, `uid`, `autostart`, `created`, `created_by`, `modified`, `modified_by`)
SELECT 'COM_GUIDEDTOURS_TOUR_WHATSNEW_6_0_TITLE', 'COM_GUIDEDTOURS_TOUR_WHATSNEW_6_0_DESCRIPTION', '["com_cpanel"]', 'administrator/index.php', 1, '*', '', 1, 'joomla-whatsnew-6-0', 1, CURRENT_TIMESTAMP(), 0, CURRENT_TIMESTAMP(), 0
WHERE NOT EXISTS (SELECT * FROM `#__guidedtours` g WHERE g.`uid` = 'joomla-whatsnew-6-0');
```
Don't insert again if the record already exists. See also PR #45972 for the reason.

Reply:
Change suggestions for the steps will follow soon.

Reply:
Hmm, for the steps it seems to be not that easy, so I would leave that as it is for now.

Reply:
Thank you Richard, I had missed https://github.com/joomla/joomla-cms/pull/45972. 
I will update our Guided Tours toolkit (which offers tour SQL export) to reflect those changes so that we build on the knowledge.

Reply:
Well, my PR was not there when you have created this one, so you haven't really missed it.